src/main/java/com/assignment/Assignment5.java

```diff
@@ -0,0 +1,10 @@
+package com.assignment;
+
+import java.util.Arrays;
+
+public class Assignment5 {
+    public static int findIndex(String indexString) {
+        String [] name = {"Paulo Dybala","Federico","Gianluigi","Ronaldo","Messi"};
```

Comment:
Why are we hardcoding this array inside this method?
Please pass this a  parameter to this method